extensions/amp-autocomplete/0.1/test-e2e/test-amp-autocomplete.js

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend('amp-autocomplete', {
+  testUrl: 'http://localhost:8000/test/manual/amp-autocomplete/amp-autocomplete.amp.html',
+  experiments: ['amp-autocomplete'],
+  // By default, the browser opens at 800x600
+  initialRect: {width: 800, height: 800},
+
+  // By default, E2E tests run in all three environments
+  // environments: ['single', 'viewer-demo', 'shadow-demo']
+}, env => {
+  let controller;
+
+  beforeEach(() => {
+    controller = env.controller;
+  });
+
+  it('<amp-autocomplete> should render correctly', async() => {
+    const autocomplete = await controller.findElement('#autocomplete');
+    const input = await controller.findElement('#input');
+    const renderedResults =
+      await controller.findElement('.i-amphtml-autocomplete-results');
+    await expect(autocomplete).not.to.be.null;
```

Comment:
`findElement` will never return `null` so you can omit these `not.to.be.null` checks. You can replace them with checks for child element counts or checking for some positive values.